packages/wdio-selenium-standalone-service/src/launcher.js

```diff
@@ -36,7 +36,15 @@ export default class SeleniumStandaloneLauncher {
          * update capability connection options to connect
          * to standalone server
          */
-        this.capabilities.forEach((cap) => Object.assign(cap, DEFAULT_CONNECTION))
+        (
+            Array.isArray(this.capabilities)
```

Comment:
Some comment as the other capability mapping

Reply:
What do you mean by that?

Reply:
I think @erwinheitzman  means that there could be a comment like:

*"In multiremote, `capabilities` is an object not an array — then, use the object's values as the array"*

So the code becomes simpler to read for new contributors who didn't know that much about Multiremote yet.